airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -120,16 +122,46 @@ def get_fields_from_schema(schema: Mapping[str, Any]) -> List[str]:
 
     @staticmethod
     def convert_schema_into_query(
-        schema: Mapping[str, Any], report_name: str, from_date: str = None, to_date: str = None, cursor_field: str = None
+        schema: Mapping[str, Any],
+        report_name: str,
+        from_date: str = None,
+        to_date: str = None,
+        cursor_field: str = None,
+        id_list: List[str] = None,
+        id_field: str = None,
+        resource_type: str = None,
+        limit: int = None,
     ) -> str:
         from_category = REPORT_MAPPING[report_name]
         fields = GoogleAds.get_fields_from_schema(schema)
+        if report_name != "change_status" and "change_status.last_change_date_time" in fields:
+            fields.remove("change_status.last_change_date_time")
```

Comment:
please add comment why we need remove `last_change_date_time` field

Reply:
Added